internal/uidriver/glfw/ui_unix.go

```diff
@@ -67,3 +68,8 @@ func (u *UserInterface) nativeWindow() uintptr {
 func (u *UserInterface) isNativeFullscreen() bool {
 	return false
 }
+
+func (u *UserInterface) setNativeCursor(shape driver.CursorShape) {
+	// TODO: Use native API in the future (issue #1571)
```

Comment:
s/issue #1571/#1571/

Reply:
So just replace (issue #1571) with that?

Reply:
With `#1571`

Reply:
I meant to eliminate the prefix `issue `.